### DISCUSSION TITLE
data() returns class "packageIQR"

### DISCUSSION THREAD

**Discussion by jimrothstein** [pharmaverse/admiral#2504] [Q&A] [answered]
I don't think this particularly important, but it did suprise me.

What is IQR?

> data(package="pharmaversesdtm") |>  class()     
>       >packageIQR

https://www.rdocumentation.org/packages/utils/versions/3.6.2/topics/data

https://search.r-project.org/CRAN/refmans/repr/html/00Index.html

**Comment by bms63** [comment]
@pharmaverse/admiral any ideas?

seems just like the default class for a package?

**Comment by bms63** [comment]
@jimrothstein do we need to chase this more or can we close this discussion?

**Reply by jimrothstein** [reply, marked as answer]
_from utils::data()_ https://stat.ethz.ch/R-manual/R-devel/library/utils/html/data.html

data() returns with object of class packageIQR

> If no data sets are specified, data lists the available data sets. For each package, it looks for a data index in the ‘Meta’ subdirectory or, if this is not found, scans the ‘data’ subdirectory for data files using list_files_with_type. The information about available data sets is returned in an object of class "packageIQR". The structure of this class is experimental. Where the datasets have a different name from the argument that should be used to retrieve them the index will have an en
